src/main/java/com/jfrog/ide/idea/configuration/ServerConfigImpl.java

```diff
@@ -451,6 +466,18 @@ public void readConnectionDetailsFromEnv() {
         }
     }
 
+    /**
+     * Read missing configuration from environment variables.
+     */
+    public void readMissingConfFromEnv() {
+        if (isBlank(getExternalResourcesRepo())) {
+            String externalResourcesRepoEnv = EnvironmentUtil.getValue(EXTERNAL_RESOURCES_REPO_ENV);
+            if (isNotBlank(externalResourcesRepoEnv)) {
+                setExternalResourcesRepo(externalResourcesRepoEnv);
+            }
```

Comment:
```suggestion
          setExternalResourcesRepo(EnvironmentUtil.getValue(EXTERNAL_RESOURCES_REPO_ENV));
```

Reply:
I did it to handle a case when the value of externalResourcesRepoEnv is an empty string.